IPython/core/display.py

```diff
@@ -622,7 +645,7 @@ def _pngxy(data):
 def _jpegxy(data):
     """read the (width, height) from a JPEG header"""
     # adapted from http://www.64lines.com/jpeg-width-height
-    
+
```

Comment:
Yeah I'm sorry if the whitespace diffs are a bit annoying. I believe it was done automatically by my editor on save (Atom). I can try to correct it if desired.